modules/environment/aws/code-services/variables.tf

```
@@ -5,3 +5,4 @@ variable "cluster" {}
 variable "openid_connect_provider_arn" {}
 variable "openid_connect_provider_url" {}
 variable "vpc_name" {}
+variable "s3-logging-id" {}
```

We should keep the variable naming consistent here and use underscores throughout 
```suggestion
variable "s3_logging_id" {}
```